pkg/store/storepb/rpc.proto

```diff
@@ -92,6 +92,8 @@ message SeriesRequest {
 
   // TODO(bwplotka): Move Thanos components to use strategy instead. Including QueryAPI.
   PartialResponseStrategy partial_response_strategy = 7;
+
+  bool no_chunk = 8;
```

Comment:
This requires a comment.

Overall are we happy with naming and this API field? Wonder how extendible this is. Overall having bool field to RPC is sometimes a [code/design smell](https://martinfowler.com/bliki/FlagArgument.html).

I think I am ok with this here as it's a caller decision to no skip chunks so maybe field should be named:

* `bool skip_chunks` or `bool discard_chunks`? (:

Reply:
I prefer `skip_chunks`. Do we need to make this as an enum? Currently I don't see some possible extensions of this field.